src/partest/scala/tools/partest/nest/FromString.scala

```diff
@@ -69,3 +70,4 @@ object FromString {
     def apply(s: String)                  = s.toInt
```

Comment:
```suggestion
    def apply(s: String)                  = Integer.parseInt(s)
```


Agree about LOC weight and such, but the lint is right - why not fix the code? It's a great example where the lint actually helps

```scala
scala> implicit val s2i: String => Int = x => x.toInt
                                              ^
       warning: Implicit resolves to enclosing value s2i
val s2i: String => Int = $Lambda/0x0000000800644a40@446e7065

scala> "3": Int

       val $line3$read: $line3.$read.INSTANCE.type = $line3.$read.INSTANCE; import $line3$read.$iw.`s2i`
                                                                                                   ^
<synthetic>:2: warning: Unused import

(To diagnose errors in synthetic code, try adding `// show` to the end of your input.)
java.lang.StackOverflowError
  at $anonfun$s2i$1(<console>:1)
  at $anonfun$s2i$1$adapted(<console>:1)
```

Reply:
OK here I'll restore it to its previous correct version, then rip it out in the other PR.

partest does no arg converting, is why it didn't matter.